scico/loss.py

```diff
@@ -13,6 +13,8 @@
 from functools import wraps
 from typing import Callable, Optional, Union
 
+import jax
```

Comment:
No other changes to this module, but now `jax` is a required import? That's rather strange.

Reply:
Yup, it's a code smell that it was never a problem before. The important line:

```python
if snp.isscalar(other) or isinstance(other, jax.core.Tracer):
```